src/nfa/operations.cc

```diff
@@ -664,11 +690,13 @@ bool Mata::Nfa::is_lang_empty(const Nfa& aut, Run* cex)
 } // is_lang_empty }}}
 
 
+//method?
```

Comment:
We agreed that functions should return new instance and methods should compute inplace, if at all possible. Therefore, this function should remain a function.